src/main/resources/org/sagebionetworks/web/client/widget/discussion/ReplyWidgetViewImpl.ui.xml

```diff
@@ -2,10 +2,17 @@
 <ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
 	xmlns:g="urn:import:com.google.gwt.user.client.ui" 
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
-	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
-	<b:Row>
-		<b:Column size="XS_12">
-			<bh:Paragraph>reply comment</bh:Paragraph>
-		</b:Column>
-	</b:Row>
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
+	xmlns:t="urn:import:org.sagebionetworks.web.client.view.bootstrap.table">
+	<bh:Div addStyleNames="margin-top-10 light-border-bottom" width="100%" pull="RIGHT" >
+		<t:Table width="100%">
+			<t:TableData>
+				<bh:Span ui:field="author"></bh:Span>
+			</t:TableData>
+			<t:TableData width="130px" addStyleNames="visible-lg visible-md">
```

Comment:
Need to autohide "created on" on small devices (current set up looks like it will hide on < 992px)?  Consider 'hidden-xs' if more appropriate.